features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/services/logout/logout.jag

```diff
@@ -0,0 +1,13 @@
+<%
+    var log = new Log("Logout Request Function");
+    var idTokenP1 = request.getCookie("AM_ID_TOKEN_DEFAULT_P1").value;
+    var idTokenP2 = request.getCookie("AM_ID_TOKEN_DEFAULT_P2").value;
+    var idToken = idTokenP1 + idTokenP2;
+    // var idToken = session.get('idToken');
+    var postLogoutRedirectURI = "https://localhost:9443/store-new/services/auth/callback";
+    var logoutState = "state_1";
```

Comment:
@CrowleyRajapakse shall we use a more readable value here.

Reply:
yes, we can make this empty also, but it's provided to check the whether the logout is successful from IDP side, only if it's successful response will have query param state with value we have sent, we can check whether this logout is initiated by us not by another party, in the doc they have used value "state_1", we can use any value as well.